bindings/pyroot/pythonizations/python/ROOT/__init__.py

```diff
@@ -72,10 +72,11 @@ def pythonization_impl(fn):
         module = importlib.import_module(pyz.__name__ + '.' + module_name)
 
 # Check if we are in the IPython shell
-try:
+if major == 3:
```

Comment:
Can you explain why this change? We can't do try except because `builtins` can also exist in Python2 but, in that case, it does not contain `__IPYTHON__`?

Reply:
Ok thanks! Can you include this explanation in the commit message?